src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> inputData) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (int i = 1; i < inputData.size(); i++) {
+            String data = inputData.get(i);
+            String[] parts = data.split(",");
+
+            if (parts.length != 3) {
+                throw new IllegalArgumentException("Incorrect input data format: " + data);
+            }
+
+            String operationCode = parts[0].trim();
+            String fruitName = parts[1].trim();

```

Comment:
why do you need to trim it?

Reply:
I use the trim() method to remove leading and trailing spaces